src/tools/pub/cmd/index.md

```diff
@@ -66,18 +66,24 @@ or any modification of the pubspec.
 
 [`pub get`](/tools/pub/cmd/pub-get)
 : Retrieves the packages that are listed as the dependencies for
-  the application. One of the most commonly used pub commands.
+  the application. If a `pubspec.lock` file already exists, fetches
+  the same version of each dependency (if missing) as listed in the
```

Comment:
This isn't *quite* correct. It'll prefer the locked version if possible, but it that's not possible—for example, because it's incompatible with a new SDK version, or a new dependency that was added—it'll choose a new version instead.